backend/src/routes/users.js

```diff
@@ -1,15 +1,92 @@
+const auth = require('../auth');
 const express = require('express');
 const router = express.Router();
 const models = require('../models');
+const db = require('../db');
+const utils = require('../utils');
+const randtoken = require('rand-token');
 
-router.get('/:id', (req, res) => {
-    res.send('returning specific user ' + req.params.id);
-});
+const getUser = async (req, res) => {
+    try {
+        const phoneNumber = req.body.phoneNumber;
+        const user = await db.users.getUser(phoneNumber);
+        res.status(200).send(user);
+    } catch (err) {
+        res.status(420).send('Failed to get user: ' + err);
+    }
+};
 
-router.get('/', (req, res) => {
-    models.users.findAll({}).then(function(users) {
+const getAllUsers = async (req, res) => {
```

Comment:
Should this kind of endpoint exist in the first place? Is there a need in the app to show all of the users? Is that even desirable that you can see all of the other users?

Reply:
would this be of any use for the application in the future?. or even for us to check all the users who are registered in the system at a certain point?.

Reply:
Since it poses a security threat, I would leave it out. We do have access to database if necessary.